tutorials/quickstart/tutorial.md

```diff
@@ -123,9 +123,105 @@ NOTE: If you forgot to detach the debugger, you'll need to click the **Continue*
+
+Another great feature of ServicePulse is the **Monitoring** view, which tracks performance statistics for all of your endpoints:
+
+![ServicePulse: Monitoring View](pulse-monitoring.png)
+
+For a more in-depth look at the monitoring capabilities, check out the [Monitoring Demo](/tutorials/monitoring-demo/), which includes a load simulator to create monitoring graphs that aren't flatlined at zero.
+
+For now, let's focus on the **Failed Messages** view. It's not much to look at right now (and that's good!) so let's generate a systemic failure:
+
+1. If you're able, undock the ServicePulse browser tab into a new window so you can better see what's going on.
+2. In the **ClientUI** window, send one message while watching the **Sales** window.
+
+Immediately, you'll see exception text flash past, culminating in an orange-colored WARN message:
+
+```
+WARN  NServiceBus.RecoverabilityExecutor Delayed Retry will reschedule message 'ea962f05-7d82-4be1-926a-a9de01749767' after a delay of 00:00:10 because of an exception:
+System.Exception: BOOM
+   at <long stack trace>
+```
+
+10 seconds later, text will flash past again, warning of a 20-second delay. 20 seconds later, the text will flash again, warning of a 30-second delay. And finally, 30 seconds after that, text will flash by again, finally ending in red ERROR message:
+
+```
+ERROR NServiceBus.RecoverabilityExecutor Moving message 'ea962f05-7d82-4be1-926a-a9de01749767' to the error queue 'error' because processing failed due to an exception:
+System.Exception: BOOM
+   at <long stack trace>
+```
+
+Once the red stack trace appears, check out the **Failed Messages** view in the **ServicePulse** window:
+
+![ServicePulse: Failed Messages View](pulse-failed-messages.png)
+
+So what happened here? The message couldn't be successfully processed during an immediate round of retries, so it delayed the message for 10 seconds to try again. After that, it could still not be processed successfully, so it delayed the message for an additional 20 seconds, and then 30 seconds, before giving up all hope and transferring the message to an **error queue**, a holding location for poison messages so that other messages behind it can still get processed successfully.
+
+Once the message enters the error queue, ServicePulse takes over, displaying all failed messages grouped by exception type and the location where it's thrown from.
+
+If you click anywhere on the exception group, it will take you to the list of exceptions within that group. This is not too interesting, since we currently only have one, but if you click again on the individual exception, you will get a rich exception detail view:
```

Comment:
``` suggestion
If you click on the exception group, it will take you to the list of exceptions within that group. This is not too interesting, since we currently only have one, but if you click again on the individual exception, you will get a rich exception detail view:
```